sdk/servicebus/azure-servicebus/tests/perf_tests/T1_legacy_tests/send_message.py

```diff
@@ -0,0 +1,25 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from ._test_base import _SendTest
+
+from azure_devtools.perfstress_tests import get_random_bytes
+
+from azure.servicebus import Message
+from azure.servicebus.aio import Message as AsyncMessage
+
+
+class LegacySendMessageTest(_SendTest):
+    def __init__(self, arguments):
+        super().__init__(arguments)
+        self.data = get_random_bytes(self.args.message_size)
+
+    def run_sync(self):
+        message = Message(self.data)
```

Comment:
Can `Message` be reused or is a new instance required for each call to `send()`?

Reply:
Customers are not likely to reuse `Message`, so our perf test should not either.